packages/react-docsite-components/package.json

```diff
@@ -47,7 +46,7 @@
     "@fluentui/set-version": "^8.1.5",
     "@fluentui/react-monaco-editor": "^1.2.80",
     "color-check": "0.0.2",
-    "markdown-to-jsx": "6.11.4",
+    "markdown-to-jsx": "^7.0.0",
```

Comment:
The changes in this package are to address a similar root cause as the react-router issue: storybook pulls in `markdown-to-jsx` version 7, which declares its own types. v8 legacy docsite uses `markdown-to-jsx` version 6 + `@types/markdown-to-jsx`. For whatever reason, yarn chose to hoist version 7 to the root, so this package's build was pulling in types for version 7.

In this case upgrading our code to version 7 was easy, so I went with that instead of adding a `nohoist` entry.

Note that the `^` dep is because this is a dependency, not devDependency.